hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.js

```diff
@@ -242,34 +244,140 @@ export const ReturnedTable = connect(returnedTableMapStateToProps, {
+                    >
+                        {isVisible ? "Hide all" : "Show all"}
+                    </Button>
+                </Container>
+            </Container>
+
+            {isVisible && (
+                <TableContainer component={Paper} elevation={3} square={true}>
+                    <Table
+                        className={styles.table}
+                        size="small"
+                        aria-label="pending table"
+                    >
+                        <TableHead>
+                            <TableRow>
+                                <TableCell className={styles.widthFixed} />
+                                <TableCell className={styles.widthHalf} align="left">
+                                    Name
+                                </TableCell>
+                                <TableCell
+                                    className={styles.widthQuarter}
+                                    align="right"
+                                >
+                                    Requested Qty
+                                </TableCell>
+                                <TableCell
+                                    className={styles.widthQuarter}
+                                    align="right"
+                                >
+                                    Granted Qty
+                                </TableCell>
+                                <TableCell className={styles.widthBuffer} />
+                            </TableRow>
+                        </TableHead>
+                        <TableBody>
+                            {items.map((row) => (
+                                <TableRow key={row.id}>
+                                    <TableCell align="left">
+                                        <img
+                                            className={styles.itemImg}
+                                            src={row.url}
+                                            alt={row.name}
+                                        />
+                                    </TableCell>
+                                    <TableCell align="left">{row.name}</TableCell>
+                                    <TableCell align="right">{row.reqQty}</TableCell>
+                                    <TableCell align="right">{row.grantQty}</TableCell>
+                                    <TableCell className={styles.widthBuffer} />
+                                </TableRow>
+                            ))}
+                        </TableBody>
+                    </Table>
+                </TableContainer>
+            )}
+        </Container>
+    );
+};
+
+const PendingTableMapStateToProps = (state) => ({
+    isVisible: isPendingTableVisibleSelector(state),
+});
+
+export const PendingTable = connect(PendingTableMapStateToProps, {
+    toggleVisibility: togglePendingTable,
+})(UnconnectedPendingTable);
+
+export const BrokenTable = ({ items, onClickViewReport, openReportAlert }) => {
```

Comment:
You called the prop `openBrokenTable` instead of `openReportAlert` on Dashboard.js, that's why the alert wasn't working. Also remove the prop `onClickViewReport` cause it's not being used.